packages/store/src/decorators/selector.ts

```diff
@@ -26,9 +26,11 @@ export function Selector(selectors?: any[]) {
               }
             );
           return memoizedFn;
-        },
-        originalFn
+        }
       };
+      // Add hidden property to descriptor
```

Comment:
Why add hidden?

Reply:
Because it was an internal trick I had to do to make the selector work independent of order.
I would not want this exposed in the public type information.